lib/package-1-12-themes.js

```diff
@@ -21,7 +21,20 @@ function Package( files, runtime ) {
 	Package1_12.apply( this, arguments );
 }
 
-extend( Package.prototype, Package1_12.prototype );
+// Copy some properties from the Package1_12's prototype
+[
+	"AUTHORS.txt",
+	"LICENSE.txt",
+	"images"
+].concat(
+	Object.keys(Package1_12.prototype)
+		.filter( function( property ) {
+			return /\.css$/.test( property );
+		} )
+).forEach( function( prop ) {
+	Package.prototype[ prop ] = Package1_12.prototype[ prop ];
+});
```

Comment:
There's something in the current code that I believe could be simpler to look and understand at a glance. Perhaps extending the Package in one step for the css ones, and then:

```js
extend( Package.prototype, {
    "AUTHORS.txt": Package1_12[ "AUTHORS.txt" ],
    "LICENSE.txt": Package1_12[ "LICENSE.txt" ],
    "images": Package1_12[ "images" ],
    "themes": function( callback ) {
    ...
```

Feel free to completely ignore this whole comment though. :)

It's up to you.